verifiednetwork-lab/challenge
# Add Ethereum Wallet Address and Display Balance

This PR adds an input field for the user's Ethereum wallet address to the registration form and updates the serializer to include the wallet address. The wallet address is now stored in the database when a user registers an account.

## Changes
[ x ] Added ethereum_wallet_address field to the User model.
[ x ] Updated the RegistrationSerializer to include the ethereum_wallet_address field.
[ x ] Modified the registration view to handle the new field.
[ x ] Updated the React registration form to include an input for the Ethereum wallet address.

## Video Overview

https://github.com/user-attachments/assets/2e9f0126-ca1b-4099-9dae-a5c7d7096616

## How to Test
 - Pull this branch.
 - Run migrations to update the database schema:
```bash
   python manage.py makemigrations
   python manage.py migrate
```

- Start the Django server:
```bash
  python manage.py runserver
```
- Navigate to the registration page on the React frontend.
- Fill in the registration form, including the Ethereum wallet address.
- Submit the form and verify that the user is registered successfully and the wallet address is stored in the database.
